### PR TITLE
[GLA-330] ✨ feat: Added doc for moderation addon - internal use only

### PR DESCRIPTION
**Pull request by impolitepanda**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new moderation functionality for transcriptions, enabling users to scan and moderate content using the **Moderation** model with bodyguard.ai.

### CODE REVIEW

File: chapters/audio-intelligence/pages/moderation.mdx [hunk 1]
```diff
@@ -0,0 +1,167 @@
+---
+title: Moderation
+description: "Find out if your transcription needs to be moderated and why, using bodyguard.ai"
```
Comment (sboudouk) [inline chapters/audio-intelligence/pages/moderation.mdx:3]: Remove any ref to bodyguard.ai

**Review by sboudouk — CHANGES_REQUESTED** [review]
Once fixed and merged, ping Amir so he can rephrase if needed.

File: chapters/audio-intelligence/pages/moderation.mdx (hunk 1 above)
Comment (impolitepanda) [reply chapters/audio-intelligence/pages/moderation.mdx]: Removed both mentions

File: chapters/audio-intelligence/pages/moderation.mdx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+---
+
+<Note>
+This feature is on **Alpha** state.
+
+We're looking for feedbacks to improve this feature, [share yours here](https://gladiaio.typeform.com/feedback?typeform-source=app.gladia.io).
+</Note>
+
+
+The **Moderation** model scans your transcription and finds out if it contains words and expressions that needs to be moderated using bodyguard.ai.
```
[the remark this answers is not in the document]
Comment (impolitepanda) [reply chapters/audio-intelligence/pages/moderation.mdx]: Fixed

**Review by sboudouk — APPROVED** [review]

**Pull request merged**